python/python/glide/async_commands/core.py

```diff
@@ -2146,6 +2147,46 @@ async def zrange_withscores(
             Mapping[str, float], await self._execute_command(RequestType.Zrange, args)
         )
 
+    async def zrangestore(
+        self,
+        destination: str,
+        source: str,
+        range_query: Union[RangeByIndex, RangeByLex, RangeByScore],
+        reverse: bool = False,
+    ) -> int:
+        """
+        Stores a specified range of elements from the sorted set at `source`, into a new sorted set at `destination`. If
+        `destination` doesn't exist, a new sorted set is created; if it exists, it's overwritten.
+
+        When in Cluster mode, all keys must map to the same hash slot.
+
+        ZRANGESTORE can perform different types of range queries: by index (rank), by the score, or by lexicographical
+        order.
+
+        See https://valkey.io/commands/zrangestore for more details.
+
+        Args:
+            destination (str): The key for the destination sorted set.
+            source (str): The key of the source sorted set.
+            range_query (Union[RangeByIndex, RangeByLex, RangeByScore]): The range query object representing the type of range query to perform.
+                - For range queries by index (rank), use RangeByIndex.
+                - For range queries by lexicographical order, use RangeByLex.
+                - For range queries by score, use RangeByScore.
```

Comment:
Is there a way to add clickable links on python docs?
```suggestion
                - For range queries by index (rank), use `RangeByIndex`.
                - For range queries by lexicographical order, use `RangeByLex`.
                - For range queries by score, use `RangeByScore`.
```

Reply:
Not as far as I can tell, I googled around and the only mention of this I can find is if you use sphinx to generate your documentation which we are not doing. Similar to above, this snippet is copy-pasted from existing docs and they do not use angle quotes

Reply:
we use links in java, wanted to see same in all languages if possible.
Does sphinx require extra syntax for links? If it is simple, we can follow that. If not ... create a task!

Reply:
Yeah it would be nice if Python supported cross-references by default, but the existing docs don't have them and aren't using sphinx